### PR TITLE
Some minor changes

### PR DESCRIPTION
**Pull request by DeepakIERStuttgart**
Hello all,

I have made some minor changes based on the comments from Prof. Blesl in all the sections. Please review and accept the changes if everything is good.

### CODE REVIEW

File: docs/model_objective/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Model Overview
 
-This section describes:
+This section provides a general introduction about the ETSAP-TIAM model, the time resolution considered and the objective function for the linear optimization model. :
```
Comment (olejandro) [inline docs/model_objective/index.md:3]: Please remove a colon at the end of this sentence.

File: docs/regional_coverage/figs/region_description_2.png
Comment (olejandro) [inline docs/regional_coverage/figs/region_description_2.png:1]: There is region-description.png already. Do we really need both figures?

File: docs/regional_coverage/index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Regional coverage
 
-This section describes:
+This section describes all the regions covered in TIAM followed by base year information. Base-year provides details regarding energy balance, emissions modelling approach and stocks. This section further describes the methodology for demand projection until 2100 and lastly possibilities of import and export between TIAM regions. :
```
Comment (olejandro) [inline docs/regional_coverage/index.md:3]: Please remove colon at the end of this paragraph.

File: docs/regional_coverage/region-description.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Region description
 
-The considered regions of TIAM can be seen in figure 1. All are listed seperatly in table 1. 
+On a spatial level, TIAM considers 16 regions, in which some regions represent countries (e.g., the US) and others are aggregated (e.g., Africa); see Figure 1. Table 1 provides further information about the TIAM regions. 
 
-![TIAM Map](./figs/region-description.png) 
+![TIAM Map](./figs/region_description_2.png)
```
Comment (olejandro) [inline docs/regional_coverage/region-description.md:5]: Please see my previous comment regarding these figures.

File: docs/model_objective/index.md (hunk 1 above)
Comment (DeepakIERStuttgart) [reply docs/model_objective/index.md:3]: Okay will do

File: docs/regional_coverage/region-description.md (hunk 3 above)
Comment (DeepakIERStuttgart) [reply docs/regional_coverage/region-description.md:5]: Sorry need to delete the previous figure

File: docs/regional_coverage/statistics/energy-balance.md [hunk 4]
```diff
@@ -2,20 +2,17 @@
 
 General Information 
 
-An energy balance provides a detailed and comprehensive overview of the energy situation in a specific country or region. Several pieces of information are contained:
+An energy balance provides a detailed and comprehensive overview of the energy situation in a specific country or region. The energy balance provides all relevant information to describe all end-use sectors (agriculture, commercial, industry, residential and transport) for an energy system model. Therefore, energy carriers, such as coal, oil and gas are covered in the energy statistics flow. Several pieces of information are contained:
```
Comment (olejandro) [inline docs/regional_coverage/statistics/energy-balance.md:5]: >Therefore, energy carriers, such as coal, oil and gas are covered in the energy statistics flow.

Please rephrase. "Therefore" seems out of place, while "energy statistics flow" is confusing.

File: docs/regional_coverage/statistics/energy-balance.md [hunk 5]
```diff
@@ -2,20 +2,17 @@
...
-
-The energy balance provides all relevant information to describe all end-use sectors (agriculture, commercial, industry, residential and transport) for an energy system model. Therefore, energy carriers, such as coal, oil and gas are covered in the energy statistics flow.
 Depending on the level of modelling, the energy balance does not support the degree on detail. Therefore, some further expert information is necessary in order to build up a full model.
 
 ### Energy Balance for TIAM
 
-The energy balances for TIAM are provided by the IEA. There is an open data energy balance, provided by the IEA, available that gives an overview of energy production and transformation:
+TIAM uses energy balance provided by the International Energy Agency (IEA). There is an open data energy balance that gives an overview of energy production and transformation:
 
 https://stats.oecd.org/BrandedView.aspx?oecd_bv_id=enestats-data-en&doi=data-00512-en
```
Comment (olejandro) [inline docs/regional_coverage/statistics/energy-balance.md:17]: Please use the following format to include this URL in the paragraph above [energy balance](URL)

File: docs/regional_coverage/statistics/index.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # Base-Year Information
 
-This section describes:
+This section describes the energy balance approch, emissions factor for different fuels and the existing capacities of technologies in the base year(stocks).:
```
Comment (olejandro) [inline docs/regional_coverage/statistics/index.md:3]: Please remove the colon.

**Review by olejandro — CHANGES_REQUESTED** [review]
Thanks @DeepakIERStuttgart. I've pointed out some minor changes required to merge this.

File: docs/regional_coverage/statistics/index.md (hunk 6 above)
Comment (DeepakIERStuttgart) [reply docs/regional_coverage/statistics/index.md]: Done

File: docs/regional_coverage/index.md (hunk 2 above)
Comment (DeepakIERStuttgart) [reply docs/regional_coverage/index.md:3]: Also done

File: docs/regional_coverage/region-description.md (hunk 3 above)
Comment (DeepakIERStuttgart) [reply docs/regional_coverage/region-description.md:5]: I see it perfectly alright when I open Visual Studio Code

Comment (olejandro) [reply docs/regional_coverage/region-description.md:5]: Why do we need "_2" in the file name? Just give the new file the same name as the old one and overwrite it. Or is there an intention to keep both files?

File: docs/regional_coverage/index.md (hunk 2 above)
Comment (olejandro) [reply docs/regional_coverage/index.md:3]: Did you remember to push this change?

File: docs/regional_coverage/statistics/energy-balance.md (hunk 4 above)
Comment (DeepakIERStuttgart) [reply docs/regional_coverage/statistics/energy-balance.md:5]: Done!

File: docs/regional_coverage/region-description.md (hunk 3 above)
Comment (DeepakIERStuttgart) [reply docs/regional_coverage/region-description.md]: Its just meant to be like a version control. So we don't loose old figures. However, if it is too confusing. Then I change it!

Comment (DeepakIERStuttgart) [reply docs/regional_coverage/region-description.md:5]: Done!

**Review by DeepakIERStuttgart — COMMENTED** [review]
I changed them all

**Review by olejandro — APPROVED** [review]

**Pull request merged**